src/moveit_visual_tools.cpp

```diff
@@ -1175,7 +1175,12 @@ bool MoveItVisualTools::publishTrajectoryPath(const std::vector<robot_state::Rob
   moveit_msgs::RobotTrajectory trajectory_msg;
   robot_trajectory->getRobotTrajectoryMsg(trajectory_msg);
 
-  return publishTrajectoryPath(trajectory_msg, shared_robot_state_, blocking);
+  // Use first trajectory point as reference state
+  moveit_msgs::RobotState robot_state_msg;
```

Comment:
what if the trajectory is empty? what will it default to? perhaps it should setToDefaultValues() in the else statement...

Reply:
If the trajectory is empty publishTrajectoryPath will fail with an error https://github.com/ros-planning/moveit_visual_tools/blob/c37438101c9397e614a53af47bc038e5a2e5e30f/src/moveit_visual_tools.cpp#L1235.